Packs/FeedTAXII/Integrations/FeedTAXII2/FeedTAXII2.py

```diff
@@ -260,10 +261,25 @@ def reset_fetch_command(client):
     )
 
 
+def is_valid_taxii_url(url: Optional[str]):
+    """
+    Checks the correctness of the url.
+    :param url: str
+    :return: boolean whether the url valid or not.
+    """
+    if url and (parse_result := urlparse(url)):
+        path = parse_result.path
+        if path and ('taxii' in path or 'taxii2' in path):
```

Comment:
The path MUST end with `/taxii/` or `/taxii2/`. This will cause a FP if the discovery URL is:
`domain.com/taxii2/bad_path/`